app/components/ListClaimConditions/ListClaimConditions.tsx

```
@@ -0,0 +1,286 @@
+      setERC20Metadata((prvmtd) => ({ ...prvmtd, [currencyAddress]: data }));
+    } catch (err) {
+      // TODO: find how an error state might be useful
+      throw new Error(
+        `Error fetching currency metadata: ${err instanceof Error ? err.message : 'Unknown error'}`,
+      );
+    } finally {
+      setIsLoadingERCMeta(false);
+    }
+  };
+
+  const updateERC20Metadata = useCallback(async () => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: fetchERC20Metadata

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)